runtime.md

```diff
@@ -21,6 +21,7 @@ The value MAY be one of:
 
     Additional values MAY be defined by the runtime, however, they MUST be used to represent new runtime states not defined above.
 * **`pid`**: (int) is the ID of the container process, as seen by the host.
+The pid MAY be excluded or set to a value indicating empty or null if it is not seen by the host. 
```

Comment:
i wonder if its useful to further clarified what it means by "pid is not seen by the host". I only understand this phrase when reading background of https://github.com/opencontainers/runtime-spec/pull/459/files.

Perhaps something like:

```
pid MAY be excluded when the container's kernel is isolated from the host / or the host does not manage the container pids
```